docs/intro.md

```diff
@@ -1,26 +1,67 @@
 ---
 slug: /
-sidebar_position: 1
+title: Welcome
+sidebar_position: 1  
 ---
 
-# Introduction
+# Welcome to Livepeer!
 
-Welcome! The pages that follow contain comprehensive documentation of the
-Livepeer ecosystem. The documentation is split into two parts: Livepeer for
-[video developers](/video-developers/overview) and Livepeer for
-[video miners](/video-miners/overview). If you are new to Livepeer and
-interested in how it works, you might want to check out the
-[Protocol Overview](/protocol/core-concepts/overview).
+The Livepeer project aims to deliver a live video streaming network protocol that is fully decentralized, highly scalable, crypto token incentivized, and results in a solution which is cheaper to an app developer than using traditional centralized live video solutions. 
 
-The documentation is an open-source community effort, so feel free to suggest
-new topics, add new how-to guides, and provide examples wherever you think it
-might be helpful. All documentation is editable via
-[GitHub](https://github.com/livepeer/livepeer-org/tree/master/docs).
+This site is organized by topics including concepts, tasks, and reference information designed for [Orchestrators](/livepeer-quick-start/video-mining-quick-start/choosing-a-role.md) on the Livepeer Network including:
 
-## Something missing? {#something-missing}
+- [Video Miners](/video-miners/core-concepts/roles-and-responsibilities#types-of-video-miners), including [Transcoders](/video-miners/core-concepts/roles-and-responsibilities#types-of-video-miners)
+- [Video Developers](/video-developers/overview)
+
+**Note** [Delegators](/protocol/core-concepts/ecosystem-participants#delegator) can also benefit from content on this site, with some information included for you as well.
```

Comment:
for some reason this sentence reads strangely for me. maybe we should omit the clause after the comma?